addOns/ascanrules/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Maintenance changes.
 - Depend on newer version of Common Library add-on.
+- Require a statistically meaningful number of requests for time-based rules using linear regression
```

Comment:
- Should be under fixed.
- Add a dot at the end.
- Could be more clear that addresses FPs.

Reply:
Updated